managers/backup-manager/DefaultBackupManager.js

```diff
@@ -3,19 +3,33 @@
 const Promise = require('bluebird');
 const catalog = require('../../broker/lib/models/catalog');
 const eventmesh = require('../../eventmesh');
-const config = require('../../common/config');
 const logger = require('../../common/logger');
 const CONST = require('../../common/constants');
 const bm = require('./');
 const BaseManager = require('../BaseManager');
 const DBManager = require('../../broker/lib/fabrik/DBManager');
-const errors = require('../../common/errors');
-const Conflict = errors.Conflict;
 
 /* jshint nonew:false */
 new DBManager(); //to start the BnRStatusPoller
 
 class DefaultBackupManager extends BaseManager {
+  constructor() {
+    super();
```

Comment:
Even though it serves the same purpose, having the registerwatcher in the constructor doesnt seem right. How about putting it in init() and calling init() explicitly by the caller?

Reply:
Also, as I mentioned earlier, we should have the provision to run multiple manager under one node process.